app/core/configs.py

```diff
@@ -11,6 +11,10 @@ class GlobalConfig(BaseModel):
         default=60.0,
         description="The interval (in seconds) at which the inference server checks for a new model binary update.",
     )
+    confident_audit_rate: float = Field(
+        default=0.001,
```

Comment:
```suggestion
        default=1e-5,
```
I think we need to set this quite a bit lower.  If somebody is running 10fps constantly we're seeing 1M iq/day, so this would escalate 1,000 images per day.  I think escalating 1 image/day if we're doing 1fps is pretty reasonable, which is about 1e-5.